egs2/msropenin/asr1/conf/tuning/train_asr_distillhubert.yaml

```diff
@@ -0,0 +1,83 @@
+# network architecture
```

Comment:
Could you try to combine the configurations? It seems that most of them are very similar.

Reply:
I have deleted a duplicate file for the conformer configuration. Rest of the files are retained as they have changes in the frontend/transformer units